### PR TITLE
Bug fix to include MT version number

### PR DESCRIPTION
**Pull request by david-i-berry**
Fixes bug where MT version number was ignored in the encoding.

### CODE REVIEW

File: csv2bufr/resources/mappings/malawi_synop_bufr.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "inputDelayedDescriptorReplicationFactor": [0,0],
     "unexpandedDescriptors": [301150,307080],
+    "masterTableVersionNumber":32,
```
Comment (tomkralidis) [inline csv2bufr/resources/mappings/malawi_synop_bufr.json:4]: We now have `masterTableVersionNumber` in two places in this file (lines 4 and 13).  Can we consolidate somehow?

File: process_station.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/bash
```
Comment (tomkralidis) [inline process_station.sh:1]: Can we add a license header?

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: csv2bufr/resources/mappings/malawi_synop_bufr.json (hunk 1 above)
Comment (david-i-berry) [reply csv2bufr/resources/mappings/malawi_synop_bufr.json]: changed for both this and unexpanded descriptors.

File: process_station.sh (hunk 2 above)
Comment (david-i-berry) [reply process_station.sh:1]: done

Comment (david-i-berry) [reply process_station.sh:1]: done

**Pull request merged**